src/pages/gift/claim/existing-account/ConnectExistingAccount.js

```diff
@@ -0,0 +1,54 @@
+import { useContext } from 'react';
+import { ClaimContext } from '../ClaimMain';
+import { Card, Row, Col } from 'react-bootstrap';
+import Button from '../../../../components/CustomButton';
+import CardButton from '../../../../components/CardButton';
+import CardHeader from '../../../../components/CardHeader';
+
+const ConnectExistingAccount = ({
+  setExistingAccountSourceHandler,
+  prevStepHandler,
+}) => {
+  return (
+    <Card.Body>
+      <CardHeader title="Connect Account" backClickHandler={prevStepHandler} />
+      <Col className="justify-content-center align-items-center">
+        <Col className="d-flex flex-column justify-content-center align-items-center pt-4">
+          <p className="text-center">
+            <span className="d-block">
+              Connect an existing Polkadot account.
+            </span>
+          </p>
+          <Button
+            variant="outline-primary"
+            onClick={() => setExistingAccountSourceHandler('ENTER')}>
```

Comment:
I have a feedback that the enter address manually might not be the most preferrable way to enter the address. does it sound good from UX perspective to move Signer and Extension buttons to the top and move enter address manually below them as a link instead of a button?!

Reply:
I think this makes sense. Added a Link-style Button like this:

<img width="699" alt="Screenshot 2021-05-19 at 17 58 59" src="https://user-images.githubusercontent.com/7072141/118845498-1d56d080-b8cc-11eb-8c11-9d8f911aec4f.png">

Reply:
Perfect!